packages/osd-test/src/legacy_opensearch/legacy_opensearch_test_cluster.js

```diff
@@ -135,10 +134,10 @@ export function createLegacyOpenSearchTestCluster(options = {}) {
     }
 
     getUrl() {
-      const parts = opensearchTestConfig.getUrlParts();
-      parts.port = port;
+      const url = opensearchTestConfig.getUrlParts();
```

Comment:
Shouldn't we just change the name of the method, if it now returns a Url instead of a parts object?

Reply:
Yes. I think we should change the function name.